index.html

```diff
@@ -1,81 +1,65 @@
+    <link rel="stylesheet" href="style.css"/>
+    <script src="https://kit.fontawesome.com/cbc2432238.js"></script>
 </head>
-
 <body>
-
-    <h1>intuit</h1> <br>
-    <p>Mint</p>
-    <p>Quickbooks</p>
-    <p>turbotax</p> <br>
-
-    <h2>Create an Intuit account</h2> <br><br>
-    <p>an account for everything intuit, including Mint.</p> <br>
-    <p>Learn more. </p> <br>
-    <p> <strong>Already have an account? <em> Signin <a href="#"></a></em></strong></p> <br>
-
-    <form>
-
-        <fieldset>
-
-            <labe>Email address</labe> <br>
-            <input type="email" placeholder="You'll verify this later"><br><br>
-
-            <label>Phone(Recommended)</label><br>
-            <input type="phone number" placeholder="Helps with lost password and security"><br><br>
-
-            <label>Password</label><br>
-            <input type="password" placeholder="Don't reuse an old password"><br><br>
-
-            <input type="Create an acccount"><br><br>
-
-
-            <p>By clicking Create account, you agree to our<br> <em>Terms <a href="#"></a></em></p>
-            and have read and acknowledged our <em>Privacy Statement <a href="#"></a></em> <br><br>
-
-
-            <p>invisible reCAPTCHA by Google Privacy Policy <a href="#"></a> and Terms of use <a href="#"></a></p>
-
-
-
-
-        </fieldset>
-
-    </form>
-
-
-    <!--Footer-->
-
-    <footer>
-
-        <small>Legal <a href="#"></a> Privacy <a href="#"></a> Security <a href="#"></a></small><br>
-        <p>2019 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered
-            trademarks of Intuit Inc.
-            Terms and conditions, features, support, pricing, and service options subject to change without notice.</p>
-
-
-
-
-    </footer>
-
-
-
-
-
-
-
+<div class="upper-nav">
+    <div class="sign-in-btn">
+        <button class="mint-mini-button"><i class="fas fa-lock"></i>  Sign In</button>
+    </div>
+    <div class="sign-in-btn">
+        <p>Already have an Intuit account?</p>
+        <p><a href="#">I forgot my User ID or Password</a></p>
+    </div>
+</div>
+<header>
+    <img src="Assets/intuit-logo.png" width="118px" height="35px"/>
+</header>
+<section class="main-content">
+    <nav>
+        <ul>
+            <li><img src="https://digitalasset.intuit.com/IMAGE/A3OSvbxLf/logo-mint-footer.png"></li>
+            <li><img src="https://digitalasset.intuit.com/IMAGE/A3IM78ua2/logo-quickbooks.png"></li>
+            <li><img src="https://digitalasset.intuit.com/IMAGE/A9pxt917g/logo-turbotax.png"></li>
+        </ul>
+    </nav>
+    <section>
+        <form>
+            <fieldset>
+
+                <h1>Create an Intuit account</h1>
+                <p>One account for everything Intuit, including Mint.</p>
+                <p><span style="font-size: 12px; color: #0077C5;">Learn more.</span></p>
+
+                <div class="form-content">
+                    <p>Email address</p>
```

Comment:
Use the Html tag label with the **for** attribute instead of a p tag. the input tag should have an id with the same name of the  **for* attribute ex: id="email" for="email"

Reply:
So we can not have any spans in the code? Are they bad practice?

Reply:
Yes you can use spans put generally for input  tags they have designated tag call **label** so this will be the appropriate way to name inputs.